internal/pkg/agent/install/install_windows.go

```diff
@@ -109,3 +121,17 @@ func serviceConfigure(ownership utils.FileOwner) error {
 	}
 	return nil
 }
+
+func isWindowsDomainUsername(username string) (bool, error) {
+	if !strings.Contains(username, `\`) {
+		// fail fast
+		return false, nil
+	}
+
+	match, err := regexp.MatchString(`^.*(\\)(.*)$`, username)
```

Comment:
Isn't this regex match equivalent to the `strings.Contains`?

Reply:
yeah basically. but i'd rather change regexp to be more strict and not allow multiple `\`s 
need to check windows reqs for domain and user names